pkg/cli/admin/release/new.go

```diff
@@ -979,6 +979,9 @@ func (o *NewOptions) extractManifests(is *imageapi.ImageStream, name string, met
 				var labels map[string]string
 				if imageConfig.Config != nil {
 					labels = imageConfig.Config.Labels
+					// when user provides an override, the platform spec of the release payload will be passed to extract and
+					// used to set the os/arch of the override image if it is a manifestList.  This is an unsupported release configuration.
+					opts.PlatformSpec = fmt.Sprintf("%s/%s", imageConfig.OS, imageConfig.Architecture)
```

Comment:
Why isn't this just setting FilterOptions on the extract command?

Reply:
right, i've updated, ptal thanks